### DISCUSSION TITLE
关于search_nodes的优化建议添加mtime

### DISCUSSION THREAD

**Discussion by ghost** [AlistGo/alist#4314] [Ideas]
对于构建备份程序来说,文件的更新时间是一个很重要的信息，然而目前仅有parent,name,is_dir,size四个字段,此表还没有unique const
对于复杂的文件树和小文件群来说检索较慢，建议添加mtime并补全索引

**Comment by seigneurcui** [comment]
强烈建议新版本，增加文件时间timestamp字段，这个信息相当重要。
